src/render/FontHandler.cpp

```diff
@@ -140,29 +146,34 @@ void FontHandler::renderMultiLineTextBox(std::vector<std::string> text, double x
     fontHeight = (m_fontMinMax[1] - m_fontMinMax[0]) * scale;
   }
 
-  double off_y = (textheight - fontHeight) / 2.0f - m_fontMinMax[0] * scale;  //Bounding box isn't centered, so we need to add fudge factor
+  double off_y = (textheight - fontHeight) / 2.0f - m_fontMinMax[0] * scale; // Bounding box isn't centered, so we need to add fudge factor
```

Comment:
There are 4 conventions in here `off_y`, `textheight`, `fontHeight` and `m_fontMinMax`. How about `off_y`, `text_height`, `font_height` and `m_font_min_max`? 😅

Reply:
Yeah, this is very old code. We are moving away from this FontHandler. I renamed the variables to snake case.